### PR TITLE
CAA-79: API redirect

### PR DESCRIPTION
**Pull request by ariessa**

### CODE REVIEW

File: coverart_redirect/request.py [hunk 1]
```diff
@@ -346,9 +350,10 @@ def handle(self, request):
             return self.handle_index()
         if entity == 'robots.txt':
             return self.handle_robots()
-
+        if entity == 'api':  
+            return self.handle_api(request)
         self.validate_entity(entity)
-
+
```
Comment (Freso) [inline coverart_redirect/request.py]: This should be a blank line. Please remove the trailing spaces. :wink:

File: coverart_redirect/request.py [hunk 2]
```diff
@@ -346,7 +350,8 @@ def handle(self, request):
             return self.handle_index()
         if entity == 'robots.txt':
             return self.handle_robots()
-
+        if entity == 'api':  
+            return self.handle_api(request)
         self.validate_entity(entity)
```
Comment (Freso) [inline coverart_redirect/request.py]: I think there should still be an empty line here, but I'll leave it to @gentlecat whether it's good enough as‐is. :)

Comment (gentlecat) [inline coverart_redirect/request.py]: I guess keeping an empty line there wound't hurt.

File: coverart_redirect/request.py [hunk 3]
```diff
@@ -244,6 +244,10 @@ def handle_robots(self):
         '''Serve up a permissive robots.txt'''
         return Response(response="User-agent: *\nAllow: /", mimetype='text/plain')
 
+    def handle_api(self, request):
+        '''Redirect to API docs at musicbrainz.org'''
+        return request.redirect (code=301, location='https://musicbrainz.org/doc/Cover_Art_Archive/API')
```
Comment (gentlecat) [inline coverart_redirect/request.py]: There should be no space before `(`. See [PEP-8](https://www.python.org/dev/peps/pep-0008/).

**Comment by gentlecat** [comment]
Does this add a redirect from https://coverartarchive.org/api/ to wikidocs?

**Comment by gentlecat** [comment]
Also, there is a conflict with `master` branch. :warning:

File: coverart_redirect/request.py (hunk 3 above)
Comment (Freso) [inline coverart_redirect/request.py]: @gentlecat Some of the surrounding functions/methods have a space in the same place, which is probably where I got the space from to begin. Just a heads up, nothing @ariessa needs to worry about. :)

**Comment by gentlecat** [comment]
Another problem is that there are 4 commits with the same comment. You might want to squash them.

**Comment by gentlecat** [comment]
Squashed into 87be594e0113a4ba08b0cba9334ec3c9498dfa75 and merged.

**Pull request closed without merging**